functions/src/api/index.ts

```diff
@@ -14,6 +16,8 @@ const options = {
 
 const swaggerSpec = swaggerJSDoc(options)
 
+app.use(cors({ origin: 'http://localhost:3000' }))
```

Comment:
do we want this one to be in production ?

Reply:
CORS should be allowed no matter the domain, so a more generic one yes.